otherlibs/systhreads/thread.mli

```diff
@@ -41,6 +42,18 @@ val id : t -> int
    is an integer that identifies uniquely the thread.
    It can be used to build data structures indexed by threads. *)
 
+exception Exit
+(** Exception raised by user code to initiate termination of the
+    current thread.
```

Comment:
In this 4.14 version, this wording might give the impression that the `exit` function raises this exception which is not the case.

Maybe:
```
  Exception that can be raised by user code to initiate termination of the
current thread.
  Compared to calling `[Thread.exit ()]`, raising an {!Thread.Exit} exception will trigger
finalizers and catch-all exception handlers. It is the recommended way to exit threads.
```

Reply:
You're absolutely right.  I improved the wording along your lines.  Thanks!